lib/plugins/aws/package/compile/events/sqs.js

```diff
@@ -22,6 +22,7 @@ class AwsCompileSQSEvents {
             batchSize: { type: 'integer', minimum: 1, maximum: 10000 },
             enabled: { type: 'boolean' },
             maximumBatchingWindow: { type: 'integer', minimum: 0, maximum: 300 },
+            functionResponseTypes: { enum: ['ReportBatchItemFailures'] },
```

Comment:
It should be `functionResponseType` (as in case of `stream` event)

Reply:
Perfect thanks

Reply:
I just checked the CloudFormation docs and it's actually plural + an array there: https://docs.aws.amazon.com/fr_fr/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventsourcemapping.html#cfn-lambda-eventsourcemapping-functionresponsetypes

<img width="733" alt="image" src="https://user-images.githubusercontent.com/720328/143230273-e9589ffe-85d9-4f61-a835-ed714b71251f.png">

Since all (most?) properties in Serverless Framework are 1:1 mapping with CloudFormation, we should probably reflect that here too?

Reply:
AWS quite often uses plural names while supporting only singular input (in form of array with single item)

In Framework, in most cases, we work then with singular names. Here we accept a string, and not an array of strings, hence singular form. Also, it reflects 1:1 how we handle this property in other `stream` events: https://github.com/serverless/serverless/blob/ce66591a688ca734eb34eff929163134d1d21ae8/lib/plugins/aws/package/compile/events/stream.js#L35

Reply:
OK that sounds good to me, as long as we use the singular form (which seems to be the case here).

That way if we ever need to support the plural form we can.